packages/@ama-sdk/showcase-sdk/src/api/store/store-api.ts

```diff
@@ -105,7 +104,7 @@ export class StoreApi implements Api {
     const options = await this.client.prepareOptions(basePathUrl, 'GET', getParams, headers, body || undefined, tokenizedOptions, metadata);
     const url = this.client.prepareUrl(options.basePath, options.queryParams);
 
-    const ret = this.client.processCall<{ [key: string]: number }>(url, options, ApiTypes.DEFAULT, StoreApi.apiName, { 200: undefined } , 'getInventory');
+    const ret = this.client.processCall<{ [key: string]: number }>(url, options, ApiTypes.DEFAULT, StoreApi.apiName, undefined, 'getInventory');
```

Comment:
why this change?

Reply:
since this is the `revivers?` parameter of the `processCall` function